ipdb/__main__.py

```diff
@@ -168,12 +168,20 @@ def get_config():
             parser.filepath = filepath
             # Users are expected to put an [ipdb] section
             # only if they use setup.cfg
-            if filepath.endswith('setup.cfg') or filepath.endswith('pyproject.toml'):
+            if filepath.endswith('setup.cfg'):
                 with open(filepath) as f:
                     parser.remove_section("ipdb")
                     read_func(f)
+            # To use on pyproject.toml, put [tool.ipdb] section
+            elif filepath.endswith('pyproject.toml'):
+                import toml
+                toml_file = toml.load(filepath)
+                if "ipdb" in toml_file["tool"]:
```

Comment:
This line here breaks the use of ipdb when `pyproject.toml` does not contain a `tool` section.  See #227.